src/widgets/partnered/partnered.scss

```diff
@@ -19,7 +19,7 @@
       justify-content: space-between;
 
       width: 100%;
-      max-width: 500px;
+      max-width: 600px;
```

Comment:
On a screen < 800 looks not very good: 
<img width="270" alt="image" src="https://github.com/user-attachments/assets/6ae79128-4142-44e6-9e45-2c6217f79ec5" />

Maybe we can add styles that would look better on a small screen
<img width="268" alt="image" src="https://github.com/user-attachments/assets/67bd8fba-483c-434e-adc8-f69dc479aaec" />